internal/scraper/config.go

```diff
@@ -0,0 +1,11 @@
+package scraper
+
+import (
+	"github.com/grafana/synthetic-monitoring-agent/pkg/pb/synthetic_monitoring"
+	bbeconfig "github.com/prometheus/blackbox_exporter/config"
+)
+
+type ConfigModule struct {
+	bbeconfig.Module
+	Traceroute synthetic_monitoring.TracerouteSettings `yaml:"traceroute,omitempty"`
```

Comment:
We discussed possibly changing this as follows, and making it anonymous, in order to promote the fields within `TracerouteSettings` up to `ConfigModule`.
```suggestion
      synthetic_monitoring.TracerouteSettings `yaml:"traceroute,omitempty"`
```
But we choose not to do this, in order to retain more similarity with how the checks are run within BlackBoxExporter.